sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuJsonToStructs.scala

```diff
@@ -33,12 +33,37 @@ case class GpuJsonToStructs(
     timeZoneId: Option[String] = None)
     extends GpuUnaryExpression with TimeZoneAwareExpression with ExpectsInputTypes
         with NullIntolerant {
+
+  // Construct a dummy json string given an input schema, for example:
+  // schema = StructType([StructField("a", StringType), StructField("b", IntegerType)])
+  // returns "{"a": "", "b": 0}"
+  private def constructEmptyRow(): String = {
```

Comment:
I would have it be a `lazy val` instead of a function to avoid building the empty row multiple times, because given a schema, this `emptyRow` is always the same for all the inputs.